kodilla-intro/src/main/java/Book.java

```diff
@@ -0,0 +1,19 @@
+public class Book {
+
+    private static Book book;
```

Comment:
po co to? Tu zawsze będzie null i twoja metoda of będzie zracała nulla. Prywatny konstruktor powinien być.